src/complex.rb

```diff
@@ -1,3 +1,19 @@
+def _zero?(obj)
+  if obj.is_a?(Float)
+    return obj == 0.0
+  elsif obj.is_a?(Integer)
+    return obj == 0
+  elsif obj.is_a?(Rational)
+    return obj.numerator == 0
+  else
+    return obj == 0
+  end
+end
+
+def _exact_zero?(obj)
+  not imaginary.is_a?(Float) and _zero?(imaginary)
+end
+
```

Comment:
:thinking: I wonder if there's a way to have these little helper methods without polluting the `main` object...?

```
bin/natalie                                                                                                              3.0.4
nat> _zero?(0)
true
nat> methods.grep(/^_/)
[:_zero?, :_exact_zero?, :__send__, :__id__]
nat> 
```

Reply:
I guess It'd be better to pollute `Complex` than to pollute main...

Reply:
I could make it `private` on `Complex`. I wish there was a way to have module-private methods.